scholly/base/src/shared/config/publications.js

```diff
@@ -9,19 +9,16 @@ const categoryType = {
   Book: 'Book',
   Other: 'Other',
 };
-const layoutOptions = {
-  ALL_PUBLICATION: 'All Publication',
-  BY_CATEGORY: 'By Category',
+const groupByOptions = {
+  NONE: 'None',
+  CATEGORY: 'Category',
```

Comment:
Just wondering, do we need this in scholly/base if they're not being used? Or just for consistency? I think the only ones being used in the layouts are the page sizes.

Reply:
I was planning to change it to a separate ticket, but Scholly will need that for my next ticket to display the publication based on the sorting/ layout options.